src/index.ts

```diff
@@ -63,6 +63,7 @@ class Storyblok {
   private resolveCounter: number;
   public relations: RelationsType;
   public links: LinksType;
+  public version: 'draft' | 'published' | undefined;
```

Comment:
@alvarosabu I think at this point you should turn this into an Enum and use it for better typings and validation (and export it so we can improve on the DX either)

Like

```ts
enum Version {
  DRAFT = 'draft',
  PUBLISHED = 'published',
}
```

this line becomes 

```ts
public version?: Version;
```

And every time we need it

```ts
this.version = config.version ?? Version.DRAFT;
```

So also no more string literals in client initialization:

```ts
client = new StoryblokClient({
  version: Version.PUBLISHED,
});
```

What do you think?

Reply:
Hi @edodusi we should avoid `enums` overall. They are considered a harmful practice on Typescript (to learn more, check resources like https://www.totaltypescript.com/why-i-dont-like-typescript-enums) 

We could replace it for a safer alternative like:

```ts
const Versions = {
  ADMIN: "admin",
  PUBLISHED: "published"
} as const;

// Convert object key in a type
type VersionsKeys = typeof Versions[keyof typeof Versions]
```

Do you consider it a merge blocking change?

Reply:
it's not a blocking change but to me that's a solid improvement, if you prefer not to use enums it's ok and you can go with objects, but since you are working on a new version think about adding it

let me know if you are willing to add this in this PR or work on this in the future, I will then approve

Reply:
Thanks for the feedback, awesome let me quickly add it and also probably export it so the user dont need to think on strings. 😉

Reply:
absolutely, they deal is not to use magic strings but properties, so we reduce errors, we have better typings, better validation, and also we could change the string value in one single place instead of searching everywhere

TBH completely I'm still kinda convinced that an enum is the correct approach for this specific case, but I trust your judgement so let's see the final implementation